site/src/pages/index.astro

```diff
@@ -397,6 +405,8 @@ import MetaLayout from "@/layouts/MetaLayout.astro";
         <dd>
           Forms in the checkout process time out after an undisclosed period of time.
         </dd>
+        <dt>2.4.3 Focus Appearance</dt>
```

Comment:
Doesn't this fail Focus Visible rather than Focus Appearance?

(Also, Focus Appearance is 2.4.13, not 2.4.3)